src/actions/userActions.ts

```diff
@@ -52,6 +54,23 @@ export const setRegistrationToken = (
     },
 })
 
+export const sendRegistrationToken = (
+    t: any,
+    registrationToken: string
+) => async (dispatch, getState) => {
+    const { user: userState } = getState()
+    const { token, userId } = userState.user
```

Comment:
this should be replace by a selector: 
const token = getTokenSelector(getState)

Reply:
even if I needed the state to also access the `userId` variable ?

Reply:
another selector with it so